api/src/Channel.h

```diff
@@ -26,10 +25,10 @@ constexpr folly::StringPiece kYouTubePlaylistService{"youtube-playlist"};
 constexpr folly::StringPiece kYouTubeService{"youtube"};
 
 constexpr std::array<folly::StringPiece, 12> kServices{
```

Comment:
Should we change this too?
```suggestion
constexpr std::array<folly::StringPiece, 10> kServices{
```

Reply:
the eyes of an eagle